clusterloader2/pkg/measurement/util/pvc_store.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+This file is copy of https://github.com/kubernetes/kubernetes/blob/master/test/utils/pod_store.go
+with slight changes regarding labelSelector and flagSelector applied and changing to PVCs.
+*/
+
+package util
+
+import (
+	"fmt"
+	"time"
+
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/apimachinery/pkg/watch"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/cache"
+)
+
+// PVCStore is a convenient wrapper around cache.Store that returns list of v1.PVC instead of interface{}.
+type PVCStore struct {
+	cache.Store
+	stopCh    chan struct{}
+	Reflector *cache.Reflector
+}
+
+// NewPVCStore creates PVCStore based on given namespace, label selector and field selector.
+func NewPVCStore(c clientset.Interface, namespace string, labelSelector string, fieldSelector string) (*PVCStore, error) {
```

Comment:
In the meantime, please use ObjectSelector introduced in https://github.com/kubernetes/perf-tests/pull/660/files (hopefully it will merge before you will read this comment).

Reply:
OK, that can be a different PR. For this one i can incorporate the object selector.